AirCasting/AirCastingApp.swift

```diff
@@ -51,6 +54,7 @@ struct AirCastingApp: App {
             RootAppView(sessionSynchronizer: sessionSynchronizer, persistenceController: persistenceController)
                 .environmentObject(authorization)
                 .environmentObject(microphoneManager)
+                .environmentObject(appState)
```

Comment:
Why making appState `environmentObject` if you don't use it anywhere?

Reply:
same: I cannot see it, maybe from earlier commit? 🙁